lib/puppet/type/firewall.rb

```diff
@@ -1033,6 +1033,20 @@ def should_to_s(value)
     newvalues(/^([0-9a-f]{2}[:]){5}([0-9a-f]{2})$/i)
   end
 
+  newproperty(:physdev_in, :required_features => :iptables) do
+    desc <<-EOS
+      Match if the packet is entering a bridge from the given interface.
+    EOS
+    newvalues(/^[a-zA-Z0-9\-\._\+]+$/)
+  end
+
+  newproperty(:physdev_out, :required_features => :iptables) do
+    desc <<-EOS
+      Match if the packet is leaving a bridge via the given interface.
```

Comment:
These docs are for `--physdev-is-out` not `--physdev-out` as per http://ipset.netfilter.org/iptables-extensions.man.html